src/portfolios/portfolio/components/shared/PortfolioSummaryPageHead.vue

```diff
@@ -163,10 +163,9 @@ export default class PortfolioSummaryPageHead extends Vue {
   }
 
   public saveTitle(): void {
-    const obj ={
-      title: this._title
+    if(hasChanges(PortfolioStore.currentPortfolio.title, this._title)) {
+      PortfolioStore.updatePortfolioTitle(this._title);
```

Comment:
need to also add old logic back in to update the `currentPortfolio` value with the new title so subsequent blurs without changes don't make unneeded API calls

Reply:
I test only the single modification before a page refresh. Didn't hit the re-tabbing to the same field without a refresh on after an initial update.